compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/diagnostics/LightTreePositioningStrategies.kt

```diff
@@ -883,6 +883,26 @@ object LightTreePositioningStrategies {
             return markElement(nodeToMark ?: node, startOffset, endOffset, tree, node)
         }
     }
+
+    @OptIn(ExperimentalStdlibApi::class)
+    val COMMAS: LightTreePositioningStrategy = object : LightTreePositioningStrategy() {
+        override fun mark(
+            node: LighterASTNode,
+            startOffset: Int,
+            endOffset: Int,
+            tree: FlyweightCapableTreeStructure<LighterASTNode>
+        ): List<TextRange> {
+            return buildList {
+                val childrenRef = Ref<Array<LighterASTNode?>>()
+                tree.getChildren(node, childrenRef)
+                for (child in childrenRef.get()) {
+                    if (child?.tokenType == KtTokens.COMMA) {
+                        add(markSingleElement(child!!, child, startOffset, endOffset, tree, node))
```

Comment:
I suggest not using `!!` operator if code can be rewritten without it.

```suggestion
                    if (child != null && child.tokenType == KtTokens.COMMA) {
                        add(markSingleElement(child, child, startOffset, endOffset, tree, node))
```

Reply:
Thanks! Sounds good!